Article/Article.js

```diff
@@ -3,27 +3,26 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement=domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton= this.domElement.querySelector('.expandButton');;
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => {this.expandArticle()});
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
```

Comment:
Try to add an `IF` conditional, where, when you open your article, the `expand` text changes to `close` and when you close it, it changes back to `expand`